setup.py

```diff
@@ -149,13 +217,14 @@ def get_extensions():
 
     sources = [os.path.join(extensions_dir, s) for s in sources]
 
-    include_dirs = [extensions_dir]
+    include_dirs += [extensions_dir]
 
     ext_modules = [
         extension(
             'torchvision._C',
             sources,
             include_dirs=include_dirs,
+            library_dirs=library_dirs,
```

Comment:
nit: this is not needed anymore

Reply:
maybe it is, given that we changed `include_dirs`, good to double-check

Reply:
I think it might be good to include if future functions require other libraries?